FirebaseFunctions/Tests/ObjCIntegration/FIRIntegrationTests.m

```diff
@@ -30,11 +31,35 @@ @interface MessagingTokenProvider : NSObject <FIRMessagingInterop>
 @implementation MessagingTokenProvider
```

Comment:
Nit (can't remember if this existed before or not): `FIR` prefix ere and for the AuthTokenProvider?